bundles/org.openhab.binding.omnikinverter/src/main/java/org/openhab/binding/omnikinverter/internal/OmnikInverterBindingConstants.java

```diff
@@ -30,7 +30,30 @@
     public static final ThingTypeUID THING_TYPE_OMNIK = new ThingTypeUID(BINDING_ID, "omnik");
 
     // List of all Channel ids
+    public static final String CHANNEL_CURRENT_PV1 = "currentPV1";
+    public static final String CHANNEL_CURRENT_PV2 = "currentPV2";
+    public static final String CHANNEL_CURRENT_PV3 = "currentPV3";
+
+    public static final String CHANNEL_VOLTAGE_PV1 = "voltagePV1";
+    public static final String CHANNEL_VOLTAGE_PV2 = "voltagePV2";
+    public static final String CHANNEL_VOLTAGE_PV3 = "voltagePV3";
+
     public static final String CHANNEL_POWER = "power";
```

Comment:
how do you handle deprecated channels?

Reply:
There is good solution until now.
For backward compatibility, you could keep it. So please check that this is the case.

Reply:
Yeah thought so, it's still functional though! Is it common to add a deprecated message in the README? And remove it once Openhab jump a major version? That does introduce tight coupling between the addons and Openhab... but I guess that's kind of inherent to a add-on :)

Reply:
In case your binding is not too much used, you could break compatibility by removing the old channel. 
All users will have to remove/create again the thing (in case it was created with UI) and then update the channel link. 
In case you want to go in this direction, you have to declare a breaking change in the release notes to let users known your breaking change.

Reply:
And if you keep it, yes you could note it as deprecated in README.

Reply:
Will add a deprecated note in the README, for now.